docs/DeveloperGuide.md

```diff
@@ -269,30 +269,299 @@ b. Require developers to download those libraries manually (this creates extra w
 
 Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (unlikely to have) - `*`
 
-
 Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
-`* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
-`* * *` | user | add a new person |
-`* * *` | user | delete a person | remove entries that I no longer need
-`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
-`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
-`*` | user with many persons in the address book | sort persons by name | locate a person easily
+`* * *`| user | add a task | record tasks that need to be done �some day�. 
+`* * *`| user |find upcoming tasks | decide what needs to be done soon. 
```

Comment:
"find" sounds like a search feature to me. If you want to have a list of tasks displayed, how about using "view" or "list"?